docs/source/tutorials/hospital_admissions_model.qmd

```diff
@@ -141,7 +143,7 @@ inf_hosp_int = deterministic.DeterministicPMF(
 )
 
 hosp_rate = metaclass.DistributionalRV(
-    dist=dist.LogNormal(jnp.log(0.05), 0.1),
+    dist=dist.LogNormal(jnp.log(0.05), jnp.log(1.1)),
```

Comment:
$\log(1.1) \approx 0.1$ - why the change?

Reply:
See https://github.com/CDCgov/multisignal-epi-inference/pull/281#discussion_r1684809312